components/wrappers/IssuesContextWrapper.js

```diff
@@ -4,8 +4,10 @@ import { useSocrata, handleIssueData } from "../utils";
 import { ISSUES_ENDPOINT } from "../settings";
 
 const STATUSES = ["needs_scoping", "backlog", "in_progress", "completed"];
+
+// The `labels` column is a comma-separated string of labels, so we use wildcard string searching
 const QUERY =
-  "$limit=100000&$where=(labels like '%Project Index%' or labels like '%Product Index%') and pipeline != 'Icebox'";
+  "$limit=100000&$where=(labels like '%Project Index%' or labels like '%Product Index%') and labels not like '%Archived Project%' and pipeline != 'Icebox'";
```

Comment:
I tested this locally with https://github.com/cityofaustin/atd-data-tech/issues/3163 and adjust **Archived Project** with **Epic**. I was able to exclude it using the **Icebox** pipeline filter so this looks like it works to me!